### PR TITLE
New Posts: PyConES 2020 C4P

### PR DESCRIPTION
**Pull request by bameda**
TODO:

- [ ] Comprobar que las cuentas de email funcionan.

### CODE REVIEW

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## How will the selection process be?
+
+1. You have to send us a link to a video of no more than 5 minutes with the talk’s summary. Please note that if you’re selected we’ll ask you to record a 15’ video that will be broadcasted during the event.
+
+2. A group of volunteers and organizers of Python España will check the videos and will make the final selection, attending interest and diversity criteria. They can contact you if some question arises or some clarification is needed.
+
+3. During August we’ll contact the selected speakers so they need to create a 15’ video with the complete talk and send it to the organization before **September 15th.**
+
+4. The week before the event we will tell you the emision slot, so you can be online to answer the question live during the 5’ after the emision.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]:
```suggestion
4. The week before the event we will tell you the emission slot, so you can be online to answer the question live during the 5’ after the emission.
```

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+</small></center>
+
+We are pleased to announce that the C4P for **PyConES 2020** *(Online version)* is open!
+
+Now you can send us your proposals for the PyConES 2020 conference and this year we will have some important changes!
+
+
+## Our Code of Conduct
+
+We want to have a welcoming and pleasant conference, with a harassment-free experience for everyone. PyConES 2020 has a mandatory [code of conduct](https://www.es.python.org/pages/codigo-de-conducta.html) for all people participating in the event.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]:
```suggestion
We want to have a welcoming and pleasant conference, with a harassment-free experience for everyone. PyConES 2020 has a mandatory [code of conduct](https://es.python.org/pages/codigo-de-conducta.html) for all people participating in the event.
```
Even if the version with www works I think the non-www one looks better :)

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## Where and when:
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]: I recommend not ending any header with `:` or ending them all with `:`. I personally prefer not using `:` in the header.

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+## What is the deadline to submit my proposal:
+
+You can submit all your proposals until 23:59 on **Sunday, July 26, 2020**.
+
+
+## Type of sessions available
+
+As per the current situation this year we’re changing the format and limiting it to just one option, the 20 minutes short talk, in order to make it easier for both speakers and attendees.
+
+So, you’ll have a 20 minutes slot divided in 15 minutes for the talk and 5 minutes for questions from the attendees..
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]: either `...` or `.` but not `..`. I think single full stop is better.

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## What can I talk about:
+
+As this year there just be a single track of talks and given that there are specific events for Data Science and ML, we decided to focus on topics that have less opportunities to be shown and that are also relevant for the Python community. We’d like to propose the following topics:
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]:
```suggestion
As this year there will just be a single track of talks and given that there are specific events for Data Science and ML, we decided to focus on topics that have less opportunities to be shown and that are also relevant for the Python community. We’d like to propose the following topics:
```

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+* **Game, Design and 3D Modelling **- Python in developing games, 3-D modelling and animation.
+
+* **Embedded Python and IoT** - MicroPython, Python on Hardware, Robotics, Arduino and Raspberry Pi.
+
+* **Sys / Dev Ops** - Testing, CI/CD, Containers, Orchestration, Logging and Monitoring.
+
+* **Networking and Security** - Network Programming, Network Security and Encryption
+
+At PyConES we have an audience with different backgrounds and diverse interests. Therefore, we accept proposals that offer an introduction for those newly initiated,  as well as those proposals that deepen in some aspect in particular for those who dominate the topic.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]:
```suggestion
At PyConES we have an audience with different backgrounds and diverse interests. Therefore, we accept proposals that offer an introduction for those newly initiated, as well as those proposals that deepen in some aspect in particular for those who dominate the topic.
```
Delete extra space.

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## How will the selection process be?
+
+1. You have to send us a link to a video of no more than 5 minutes with the talk’s summary. Please note that if you’re selected we’ll ask you to record a 15’ video that will be broadcasted during the event.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]: Perhaps include a link to the form submit the proposal video from this paragraph, so people don't mistakenly use the contact@... mail to do that.

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
+Title: PyConES 2020 Call for Papers is open!!!
+Date: 2020-06-15
+Category: PyConES
+Summary: We are pleased to announce that the C4P for PyConES 2020 is open! Now you can send us your proposals for the PyConES 2020 conference and this year we will have some important changes!
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]: This English version doesn't say anything about "Online edition" in the title.

File: content/2020-06-05-pycones-2020-c4p-es.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
+Title: El Call for Papers para PyConES 2020 está abierto!!!
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-es.md]:
```suggestion
Title: ¡Abierto el Call for Proposals para PyConES 2020!
```

File: content/2020-06-05-pycones-2020-c4p-en.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
+Title: PyConES 2020 Call for Papers is open!!!
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-en.md]:
```suggestion
Title: PyConES 2020 Call for Proposals is open!!!
```
I think "Proposals" makes more sense than "Papers" for the kind of submission that we ask for.

File: content/2020-06-05-pycones-2020-c4p-es.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+Date: 2020-06-15
+Category: PyConES
+Summary: ¡Nos complace anunciar que el C4P para *PyConES 202V0 (Online edition)* está abierto! Ha llegado el momento de que nos envíes tus propuestas para la conferencia PyConES 2020. Lee con atención ya que este año presentaremos algunos cambios importantes.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-es.md]:
```suggestion
Summary: ¡Nos complace anunciar que el C4P para *PyConES 2020 (edición Online)* está abierto! Ya nos puedes enviar tus propuestas para la conferencia. Lee con atención ya que este año hay algunos cambios importantes.
```

File: content/2020-06-05-pycones-2020-c4p-es.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+<center><small>
+ NOTE: If you don’t speak Spanish [go to the ENGLISH version]({filename}/2020-06-05-pycones-2020-c4p-en.md)
+</small></center>
+
+¡Nos complace anunciar que el C4P para **PyConES 2020** *(edición Online)* está abierto!
+
+Ha llegado el momento de que nos envíes tus propuestas para la conferencia PyConES 2020. Lee con atención ya que este año presentaremos algunos cambios importantes.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-es.md]:
```suggestion
Ya nos puedes enviar tus propuestas para la conferencia. Lee con atención ya que este año hay algunos cambios importantes.
```

File: content/2020-06-05-pycones-2020-c4p-es.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+
+## Nuestro código de conducta
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-es.md]:
```suggestion
## Código de conducta
```

File: content/2020-06-05-pycones-2020-c4p-es.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+* **Sys / DevOps** - Herramientas de desarrollo y automatización, tests, CI / CD, contenedores, orquestación, registro y monitorización,…
+
+* **Redes y seguridad** -  Programación de redes, seguridad de redes, cifrado,...
+
+En PyConES tenemos un público con intereses muy variados así que aceptamos propuestas que ofrezcan una introducción para aquellas personas recién iniciadas en cualquiera de estos temas, como aquellas que deseen profundizar en algún aspecto en concreto para quien domina la materia.
+
+
+## Cómo será el proceso de selección
+
+1. Nos tienes que mandar un enlace a un vídeo (que puede ser privado) de no más de 5 minutos con el resumen de la charla. Ten en cuenta que, si te seleccionamos, te pediremos luego que antes de la conferencia graves y envíes un vídeo de 15’ que será emitido durante la conferencia.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-es.md]:
```suggestion
1. Nos tienes que mandar un enlace a un vídeo (que puede ser privado) de no más de 5 minutos con el resumen de la charla. Ten en cuenta que, si te seleccionamos, te pediremos luego que antes de la conferencia grabes y envíes un vídeo de 15’ que será emitido durante la conferencia.
```

File: content/2020-06-05-pycones-2020-c4p-es.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+
+2. un grupo formado por personas voluntarias de la Asociación Python España junto con el equipo organizador el que hará el visionado de vídeos y hará la selección final atendiendo a criterios de interés y diversidad, pudiendo contactar contigo si surge alguna duda o fuese necesaria alguna aclaración.
```
Comment (dukebody) [inline content/2020-06-05-pycones-2020-c4p-es.md]:
```suggestion
2. Un grupo formado por personas voluntarias de la Asociación Python España junto con el equipo organizador revisará los vídeos y hará la selección final atendiendo a criterios de interés y diversidad, pudiendo contactar contigo si surge alguna duda o fuese necesaria alguna aclaración.
```

**Review by dukebody — CHANGES_REQUESTED** [review]
Principalmente correcciones para hacer el texto más natural y ortográficas/gramaticales.

File: content/2020-06-05-pycones-2020-c4p-en.md (hunk 4 above)
Comment (bameda) [reply content/2020-06-05-pycones-2020-c4p-en.md]: Is a typo, It should be `.`

File: content/2020-06-05-pycones-2020-c4p-en.md (hunk 3 above)
Comment (bameda) [reply content/2020-06-05-pycones-2020-c4p-en.md]: :+1:

**Comment by bameda** [comment]
Thanks, @dukebody, for your review

**Pull request merged**